manifests/docker.pp

```diff
@@ -4,5 +4,53 @@
 #
 # @example
 #   include hdm::docker
+#
+# @api private
+#
 class hdm::docker {
+  if $hdm::manage_docker {
+    include docker
+  }
+
+  $directories = [
+    $hdm::hdm_path,
+    "${hdm::hdm_path}/certs",
+    "${hdm::hdm_path}/db",
+  ]
+  file { $directories:
+    ensure => directory,
+  }
+  file { "${hdm::hdm_path}/hdm.yml":
+    ensure  => file,
+    content => epp('hdm/hdm.yml.epp'),
+  }
+  file { "${hdm::hdm_path}/database.yml":
+    ensure  => file,
+    content => epp('hdm/database.yml'),
+  }
+  file { "${hdm::hdm_path}/db/development.sqlite3":
+    ensure => file,
+  }
+
+  docker::image { 'ghcr.io/betadots/hdm':
+    image_tag => $hdm::version,
+  }
+
+  docker::run { 'hdm':
+    image            => "ghcr.io/betadots/hdm:${hdm::version}",
+    env              => [
+      "TZ=${facts['timezone']}",
+      "RAILS_DEVELOPMENT_HOSTS=${facts['networking']['fqdn']}",
```

Comment:
Change: maybe make this configureable. so if fqdn of the vm is not the desired name to use in reverse proxy?